gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/pom.xml

```diff
@@ -459,6 +459,22 @@
                 </dependency>
             </dependencies>
         </profile>
+
+        <profile>
```

Comment:
I don't know the implementation of this dependency, but I guess it requires some specific configuration to be activated or maybe a specific license?

Because it will be embedded in the default bundle.

Reply:
I made this on purpose to include this library in the default bundle when packaging it (but it is not a plugin). 
There is no license nor specific configuration required.